sqlmesh/core/model/definition.py

```diff
@@ -58,6 +58,11 @@
 else:
     from typing_extensions import Literal
 
+if sys.version_info >= (3, 11):
+    pass
+else:
+    pass
```

Comment:
let's get rid of this?

Reply:
Well spotted, the linter removed the unused imports when I refactored and I didnt notice the conditionals were still there